src/classes/SshSessionHandler.ts

```diff
@@ -1,36 +1,45 @@
-import type { SFTPStream } from 'ssh2-streams';
 import { SftpStreamHandler } from './SftpStreamHandler';
 import { logger } from '../logger';
+import type { SFTPWrapper } from 'ssh2';
 
 export class SshSessionHandler {
+  private readonly authToken: string;
+
+  public constructor(authToken: string) {
+    this.authToken = authToken;
+  }
+
   /**
    * See: Session Events (sftp)
    * https://github.com/mscdex/ssh2#session-events
    */
   public onSftp = (
-    accept: () => SFTPStream,
+    accept: () => SFTPWrapper,
   ): void => {
     logger.verbose('SFTP requested');
-    const sftp = accept();
-    const sftpStreamHandler = new SftpStreamHandler();
-    sftp.on('OPEN', sftpStreamHandler.openHandler);
-    sftp.on('READ', sftpStreamHandler.readHandler);
-    sftp.on('WRITE', sftpStreamHandler.writeHandler);
-    sftp.on('FSTAT', sftpStreamHandler.fstatHandler);
-    sftp.on('FSETSTAT', sftpStreamHandler.fsetStatHandler);
-    sftp.on('CLOSE', sftpStreamHandler.closeHandler);
-    sftp.on('OPENDIR', sftpStreamHandler.openDirHandler);
-    sftp.on('READDIR', sftpStreamHandler.readDirHandler);
-    sftp.on('LSTAT', sftpStreamHandler.lstatHandler);
-    sftp.on('STAT', sftpStreamHandler.statHandler);
-    sftp.on('REMOVE', sftpStreamHandler.removeHandler);
-    sftp.on('RMDIR', sftpStreamHandler.rmDirHandler);
-    sftp.on('REALPATH', sftpStreamHandler.realPathHandler);
-    sftp.on('READLINK', sftpStreamHandler.readLinkHandler);
-    sftp.on('SETSTAT', sftpStreamHandler.setStatHandler);
-    sftp.on('MKDIR', sftpStreamHandler.mkDirHandler);
-    sftp.on('RENAME', sftpStreamHandler.renameHandler);
-    sftp.on('SYMLINK', sftpStreamHandler.symLinkHandler);
+    const sftpStream = accept();
```

Comment:
You know what -- I think all this should be `sftpSession` / `SftpSessionHandler` to mirror `SshSessionHandler`.